File: rvfilter/app/src/main/java/com/rvfilter/widget/filter/FilterBean.java
package com.rvfilter.widget.filter;

/**
 * by y on 2017/4/24.
 */

public class FilterBean {
    public int type;
    public String titleType;
    public String content;
    public boolean isSelect;

    public FilterBean(int type) {
        this.type = type;
    }

    public FilterBean(int type, String titleType) {
        this.type = type;
        this.titleType = titleType;
    }

    public FilterBean(int type, String titleType, String content) {
        this.type = type;
        this.titleType = titleType;
        this.content = content;
    }

    public FilterBean(int type, String titleType, String content, boolean isSelect) {
        this.type = type;
        this.titleType = titleType;
        this.content = content;
        this.isSelect = isSelect;
    }
}

File: utils_rvfilter/src/main/java/sample/util/develop/android/rv/filter/widget/filter/FilterBean.kt
package sample.util.develop.android.rv.filter.widget.filter

class FilterBean {
    var type: Int = 0
    var titleType: String = ""
    var content: String = ""
    var isSelect: Boolean = false

    constructor(type: Int) {
        this.type = type
    }

    constructor(type: Int, titleType: String) {
        this.type = type
        this.titleType = titleType
    }

    constructor(type: Int, titleType: String, content: String) {
        this.type = type
        this.titleType = titleType
        this.content = content
    }

    constructor(type: Int, titleType: String, content: String, isSelect: Boolean) {
        this.type = type
        this.titleType = titleType
        this.content = content
        this.isSelect = isSelect
    }
}
